From 2ffed8fbb6464f675bc4e8f3c60a4cb27e7e9fbf Mon Sep 17 00:00:00 2001
From: Paul Keith <javelinanddart@gmail.com>
Date: Wed, 28 Mar 2018 19:52:29 +0200
Subject: [PATCH 06/60] fs: sdfat: Add MODULE_ALIAS_FS for supported
 filesystems

* This is the proper thing to do for filesystem drivers

Change-Id: I109b201d85e324cc0a72c3fcd09df4a3e1703042
Signed-off-by: Paul Keith <javelinanddart@gmail.com>
---
 fs/sdfat/sdfat.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/fs/sdfat/sdfat.c b/fs/sdfat/sdfat.c
index c1a2c2ec1b67..0fe2a740f2b7 100644
--- a/fs/sdfat/sdfat.c
+++ b/fs/sdfat/sdfat.c
@@ -5070,6 +5070,7 @@ static struct file_system_type sdfat_fs_type = {
 #endif /* CONFIG_SDFAT_DBG_IOCTL */
 	.fs_flags    = FS_REQUIRES_DEV,
 };
+MODULE_ALIAS_FS("sdfat");
 
 #ifdef CONFIG_SDFAT_USE_FOR_EXFAT
 static struct file_system_type exfat_fs_type = {
@@ -5083,6 +5084,7 @@ static struct file_system_type exfat_fs_type = {
 #endif /* CONFIG_SDFAT_DBG_IOCTL */
 	.fs_flags    = FS_REQUIRES_DEV,
 };
+MODULE_ALIAS_FS("exfat");
 #endif /* CONFIG_SDFAT_USE_FOR_EXFAT */
 
 #ifdef CONFIG_SDFAT_USE_FOR_VFAT
@@ -5097,6 +5099,7 @@ static struct file_system_type vfat_fs_type = {
 #endif /* CONFIG_SDFAT_DBG_IOCTL */
 	.fs_flags    = FS_REQUIRES_DEV,
 };
+MODULE_ALIAS_FS("vfat");
 #endif /* CONFIG_SDFAT_USE_FOR_VFAT */
 
 static int __init init_sdfat_fs(void)
-- 
2.17.1

